src/plugins/db/index.ts

```diff
@@ -12,35 +13,32 @@ import {
   createTimestampWithTimeZoneTypeParser,
   createTimestampTypeParser,
 } from './parsers';
-import * as user from './queries/user-queries';
-import * as action from './queries/action-queries';
-import * as actionType from './queries/action-type-queries';
-import * as feedItem from './queries/feed-item-queries';
-import * as comment from './queries/comment-queries';
+import actions, { Queries } from '../../queries';
 import { transformNameInterceptors } from './utils';
 import config from '../../config';
 
+let pool: DatabasePoolType;
+export function getPool(): DatabasePoolType {
```

Comment:
Inconsistent function declarations. I've previously used const and arrow style. Personally, that is my preference, but consistency is more important. We should pick 1 and stick with it.

Reply:
This is one of the few cases where I don't think consistency needs to be strictly enforced. `const` and arrow styles can be good when you need a light utility function, but they're unnamed. If an error occurs, the stack contains a lot of `<anonymous>` and debugging becomes harder. If we have to stick with one I'd vote for functions almost purely based on the stack trace naming, but I wouldn't start extensively refactoring existing code because of that.